main.tex

```diff
@@ -624,32 +665,37 @@
 
 \begin{enumerate}[label=(\arabic*)]
 
-\item  An Owner or Occupier must not paint, finish, mark, damage, deface or otherwise alter the exterior of any Building or improvement forming part of any Lot or Common Property.
+\item  An Owner or Occupier must not paint, finish, mark, damage, deface or otherwise alter the exterior of any Building or improvement forming part of any Lot or Common Property. This ban does not extend to the growth of plants.
 
 \item  An Owner or Occupier must not damage, drive nails or screws to any Common Property walls, floors, ceilings or alter a structure that forms part of the Common Property.
 \newpage
 
 
 
-\item  An Owner or Occupier must promptly notify the Owners Corporation of any accident they are aware of which occurred in the Building, on a Lot or Common Property, noting that the details of the incident may be required in writing.
```

Comment:
If you drop your keys and pick them back up, you need to notify the Owners Corporation.